bears/css/CSSAutoPrefixBear.py

```diff
@@ -0,0 +1,16 @@
+from coalib.bearlib.abstractions.Lint import Lint
+from coalib.bears.LocalBear import LocalBear
+
+
+class CSSAutoPrefixBear(Lint, LocalBear):
+    executable = "postcss"
```

Comment:
> It looks as if postcss is the executable and autoprefixer is a plugin for it.

That *is* indeed the case. PostCSS comes with a CLI, hence it's the executable and supports many plugins which can be installed using `npm` and used with `--use` argument. https://github.com/postcss/postcss/blob/master/docs/plugins.md